index2.html

```diff
@@ -0,0 +1,1253 @@
+    <i>URI</i>, <i>Smart Poster</i> (containing a URI or other
+    data and possible actions).
+  </p>
+  <p>An <dfn>NFC handover</dfn> defines NFC Forum Well Known Types and the
+    corresponding message structure that allows negotiation and activation of
+    an alternative communication carrier, such as Bluetooth or WiFi.
+    The negotiated communication carrier would then be used (separately) to
+    perform certain activities between the two devices, such as sending photos
+    to the other device, printing to a Bluetooth printer or streaming video to
+    a television set.
+  </p>
+  <p>
+    The term <dfn>sufficient permission</dfn> in this document means that a
+    <a>web app</a> which is invoking the methods of this API has been checked to
+    comply with the security policies set by the underlying platform and API
+    implementation in the moment and context of invoking the API method.
+  </p>
+  <p>
+    The term
+    <a href="http://www.w3.org/TR/2011/WD-html5-20110113/urls.html#document-base-url">
+    <dfn>document base URL</dfn></a> is defined in [[!HTML5]].
+  </p>
+  <p>
+    The term <dfn>URL scope</dfn> is introduced in this document for NFC
+    operations as a sub-string match to the <a>web app</a>'s
+    document URL, which includes the domain of the calling <a>web app</a>.
+    The URL scope is stored in the <var>identifier</var> field of an
+    <a>NDEF record</a>.
+  </p>
+</section> <!-- Terminology -->
+
+<!-- - - - - - - - - - - - - - -  Introduction  - - - - - - - - - - - - - - -->
+<section class="informative"> <h2>Introduction</h2>
+  <p>
+    There are three groups of application scenarios for NFC:
+    <ul>
+      <li>
+        Hold a device close to a passive wireless tag (which could be in the
+        form of a plastic card) to read and write or overwrite data (in the
+        case the tag is not read-only).
+      </li>
+      <li>
+        Hold two powered devices, e.g. phones or tablets, close to each other
+        in order to push data from one to the other, or to initiate a connection
+        using another wireless carrier such as Bluetooth or WiFi.
+      </li>
+      <li>Card emulation
+       <ol>
+        <li>
+          With a secure element: for payments by holding your phone close to a
+          point-of-sales terminal, instead of swiping a payment card.
+        </li>
+        <li>With host card emulation: for allowing use-cases like using a phone
+          as a hotel room key card.
+        </li>
+       </ol>
+      </li>
+    </ul>
+  </p>
+  <p>
+    NFC works using magnetic induction, meaning that the reader will emit a
+    small electric charge which then creates a magnetic field. This field powers
+    the passive device which turns it in to electrical impulses to communicate
+    data. Thus, when the devices are within range, a read is always performed
+    (see NFC Analog Specification and NFC Digital Protocol, NFC Forum, 2006).
```

Comment:
There is no direct link. The NFC specifications are available for download on registration. It is enough to give the spec name.